src/main/java/xbot/common/controls/actuators/wpi_adapters/PWMWPIAdapter.java

```diff
@@ -10,7 +10,7 @@
 
 public class PWMWPIAdapter extends XPWM
 {
-    private PWM pwm;
+    private final PWM pwm;
```

Comment:
it's a good idea, but just curious why it was needed?

Reply:
Since I was changing the file I just fixed the warnings while I was at it.